content/docs/tutorials/getting-started-with-trust-manager/README.md

```diff
@@ -0,0 +1,600 @@
+---
+title: Managing Public Trust in Kubernetes with Trust Manager
+description: Learn how to deploy and configure trust-manager to automatically distribute your approved Public CA configuration to your entire Kubernetes cluster.
+---
+
+*Last Verified: 19 June 2023*
+
+In this tutorial we will walk through how we can use 
+[trust-manager](https://cert-manager.io/docs/projects/trust-manager/) to
+distribute publicly trusted Certificate Authority (CA) certificates inside
+a Kubernetes cluster. Once distributed we will also show:
+
+- How you can automatically reload applications when your trust bundle changes
+- How you can enforce applications to use your distributed CA bundle
+
+From there we will use a simple `curl` pod to show how to automatically mount
+the trusted CA `Bundle`, so it can be used without having to configure curl
+manually. This mimics how an application would not need any additional
+configuration to make use of your trusted CA certificates bundle.
+
+In this tutorial we will be limiting the scope of our changes to only impact
+the `team-a` namespace. To get the most out of these features you will want to
+remove this limitation.
+
+> **Note:** All resources provided are demonstrative and should be reviewed 
+  properly before using in production environments.
+
+## Prerequisites
+
+**💻 Software**
+
+1. [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl): The Kubernetes
+command-line tool which allows you to configure Kubernetes clusters.
+1. [helm](https://helm.sh/): A package manager for Kubernetes.
+1. [yq](https://github.com/mikefarah/yq#install): A command line tool for
+parsing YAML with helpful coloring.
+
+## Distribute Public CA Trust
+
+Let us first setup trust-manager and have our public CAs distributed to our
+demo namespace: `team-a`.
+
+### Setup Application & Bundle
+
+1) Ensure you have [trust-manager](../../projects/trust-manager/README.md#installation) installed. If not simply use:
+
+    ```shell
+    helm repo add jetstack https://charts.jetstack.io
+    helm upgrade -i -n cert-manager cert-manager jetstack/cert-manager --set installCRDs=true --wait --create-namespace
+    helm upgrade -i -n cert-manager trust-manager jetstack/trust-manager --wait
+    ```
+
+1) Create your first `Bundle` resource including only Public CA certificates
+
+    ```yaml file=./trust/bundle-public.yaml
+    ```
+
+    ```shell
+    kubectl apply -f - <<EOF
+    apiVersion: trust.cert-manager.io/v1alpha1
+    kind: Bundle
+    metadata:
+      name: public-bundle
+    spec:
+      sources:
+      - useDefaultCAs: true
+      target:
+        configMap:
+          key: "ca-certificates.crt"
+        namespaceSelector:
+          matchLabels:
+            trust: enabled
+    EOF
+    ```
+
+1) Let's create a namespace where our application will run:
+
+    ```shell
+    kubectl apply -f - <<EOF
+    apiVersion: v1
+    kind: Namespace
+    metadata:
+      labels:
+        kubernetes.io/metadata.name: team-a
```

Comment:
I think this is just because I got the namespace out to yaml and k8s does this automagically I think:

```shell
> k get namespace app-team-2 -o yaml 
apiVersion: v1
kind: Namespace
metadata:
  creationTimestamp: "2023-05-19T08:46:46Z"
  labels:
    kubernetes.io/metadata.name: app-team-2
  name: app-team-2
  resourceVersion: "536456"
  uid: 19ca8325-5090-4561-8474-612ccaa18cb4
spec:
  finalizers:
  - kubernetes
status:
  phase: Active
```

But I can edit that to remove it. It's not needed for this tutorial

Reply:
Removed that label as not needed in these docs.